scripts/speaker_tasks/filelist_to_manifest.py

```diff
@@ -30,21 +30,21 @@
 This scipt converts a filelist file where each line contains  
 <absolute path of wav file> to a manifest json file. 
 Optionally post processes the manifest file to create dev and train split for speaker embedding 
-training, also optionally chunk an audio file in to segments of random DURATIONS and create those
+training, also optionally segment an audio file in to segments of random DURATIONS and create those
 wav files in CWD. 
 
-While creating chunks, if audio is not sampled at 16Khz, it resamples to 16Khz and write the wav file.
+While creating segments, if audio is not sampled at 16Khz, it resamples to 16Khz and write the wav file.
```

Comment:
This is not the most critical point, but it should be "kHz" or "KHz" (kHz is more popular). H is the only one in capital. This change should be made for all the files.